fixtures/datagen/fakerize.py

```diff
@@ -445,10 +458,13 @@ def make_verified(vasp, idx, template="fixtures/datagen/templates/verified.json"
         ["administrative", "technical"]
     )  # billing always unverified for demo purposes
     record["contacts"][other] = make_person(vasp, token=other+"_token", rng=rng_person)
-    record = synthesize_secrets(record)
     common_name = "trisa." + vasp.lower().split()[0]
     record["common_name"] = common_name + ".io"
     record["identity_certificate"]["subject"]["common_name"] = common_name + ".io"
+    for cert in record["signing_certificates"]:
+        cert["subject"]["common_name"] = common_name + ".io"
+    rng_cert = random.Random(vasp+"cert")
+    record = synthesize_secrets(record, rng=rng_cert)
```

Comment:
@DanielSollis Not sure if you've seen this, but this is the script we run when we need to change the static fixtures that are committed as a .tgz to the repo.